CantoboardFramework/Info.plist

```diff
@@ -18,5 +18,9 @@
 	<string>1.0</string>
 	<key>CFBundleVersion</key>
 	<string>$(CURRENT_PROJECT_VERSION)</string>
+	<key>UIAppFonts</key>
+	<array>
+		<string>CustomFont.ttf</string>
```

Comment:
Is it possible to achieve the same goal without using custom font? Using custom font increases memory usage a lot, and keyboard extension could only use up to 33MB or 66MB (depending on iOS version).

Reply:
This is only a temporary solution.
Although I know it’s bad to use it in production, the size of the font file itself is only 9 KB, and I think it won’t significantly affect the memory usage.